.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bquorning
```

Comment:
Github team with Gabe, you and me?

Reply:
It seems a zendesk github team would not be mention-able by non-zendesk committees. I may be wrong, but that was the reason.

Reply:
[![](https://help.github.com/assets/images/help/teams/select-and-confirm-new-visibility.png)](https://help.github.com/articles/changing-team-visibility/)

:sad_panda: